src/main/proto/ReportService.proto

```diff
@@ -164,6 +164,12 @@ message getAllJiraRequest{
 message getAllJiraResponse{
     repeated JiraTickets records = 1;
 }
+message ignoreTicketCreationRequest{
```

Comment:
I see other messages have a JIRA word in their name, but this one does not. Not sure if it's good for consistency.

Also not sure it's the best idea to have JIRA specific messages.  What if we add trello integration, are we going to have another 10 trello specific messages?